src/record.js

```diff
@@ -44,7 +44,7 @@ class Record {
    */
   async decryptData(privKey, uriResolver) {
     const ciphertext = await uriResolver(this.dataUri);
-    const plaintext = await _util.decrypt(privKey, ciphertext);
+    const plaintext = _util.decrypt(privKey, ciphertext);
```

Comment:
why was the await there?

Reply:
Because the old encryption method was asynchronous